lane_mate_frontend/src/CarrierTable.js

```diff
@@ -7,92 +7,74 @@ import TableHead from "@mui/material/TableHead";
+    }
+  };
 
-export default function CarrierTable() {
   return (
-    <TableContainer component={Paper}>
-      <Table sx={{ minWidth: 650 }} aria-label="simple table">
-        <TableHead>
-          <TableRow>
-            <TableCell>Carrier Name</TableCell>
-            <TableCell align="right">Phone Number</TableCell>
-            <TableCell align="right">Dispatch Email</TableCell>
-            <TableCell align="right">Contact Name</TableCell>
-            <TableCell align="right">Rate</TableCell>
-            <TableCell align="right">Notes</TableCell>
-          </TableRow>
-        </TableHead>
-        <TableBody>
-          {rows.map((row) => (
-            <TableRow
-              key={row.carrierName}
-              sx={{ "&:last-child td, &:last-child th": { border: 0 } }}
-            >
-              <TableCell component="th" scope="row">
-                {row.carrierName}
-              </TableCell>
-              <TableCell align="right">{row.phoneNumber}</TableCell>
-              <TableCell align="right">{row.dispatchEmail}</TableCell>
-              <TableCell align="right">{row.contactName}</TableCell>
-              <TableCell align="right">{row.rate}</TableCell>
-              <TableCell align="right">{row.notes}</TableCell>
-            </TableRow>
-          ))}
-        </TableBody>
-      </Table>
-    </TableContainer>
+    <div>
+      <div>
+        <TableContainer component={Paper}>
+          <Table sx={{ minWidth: 650 }} aria-label="simple table">
+            <TableHead>
+              <TableRow>
+                <TableCell>Carrier Name</TableCell>
+                <TableCell align="right">Phone Number</TableCell>
+                <TableCell align="right">Dispatch Email</TableCell>
+                <TableCell align="right">Contact Name</TableCell>
+                <TableCell align="right">Rate</TableCell>
+                <TableCell align="right">Notes</TableCell>
+              </TableRow>
+            </TableHead>
+            <TableBody>
+              {rows.map((row) => (
+                <TableRow
+                  key={row.carrierName}
+                  sx={{ "&:last-child td, &:last-child th": { border: 0 } }}
+                >
+                  <TableCell component="th" scope="row">
+                    {row.carrierName}
+                  </TableCell>
+                  <TableCell align="right">{row.phoneNumber}</TableCell>
+                  <TableCell align="right">{row.dispatchEmail}</TableCell>
+                  <TableCell align="right">{row.contactName}</TableCell>
+                  <TableCell align="right">{row.rate}</TableCell>
+                  <TableCell align="right">{row.notes}</TableCell>
+                </TableRow>
+              ))}
+            </TableBody>
+          </Table>
+        </TableContainer>
+      </div>
+      <div>
+        <CarrierForm
+          createNewEntryOnSubmit={createNewEntryOnSubmit}
+          quoteRequestDefined={quoteRequestDefined}
+        />
```

Comment:
Hide table until `quoteRequestDefined` is true

```suggestion
        {quoteRequestDefined && (
          <CarrierForm
            createNewEntryOnSubmit={createNewEntryOnSubmit}
          />
        )}
```